Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION := $(shell pulumictl get version)
+VERSION := 1.6.1
```

Comment:
Hard coding this for now. Otherwise, if we regenerate the SDK to check-in, it's going to be 1.6.0+dirty. We need to rework this to use the same approach as other providers.